iidm/iidm-criteria/src/main/java/com/powsybl/iidm/criteria/duration/EqualityTemporaryDurationCriterion.java

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.iidm.criteria.duration;
+
+/**
+ * Describes equality temporary duration criterion
+ *
+ * @author Sophie Frasnedo {@literal <sophie.frasnedo at rte-france.com>}
+ */
+public final class EqualityTemporaryDurationCriterion extends AbstractTemporaryDurationCriterion {
+    private final int durationEqualityValue;
+
+    public EqualityTemporaryDurationCriterion(int durationEqualityValue) {
+        if (durationEqualityValue <= 0) {
+            throw new IllegalArgumentException("Invalid value (must be > 0).");
+        }
+        this.durationEqualityValue = durationEqualityValue;
```

Comment:
is this in seconds ? I think it should be indicated somewhere

Reply:
I added a Javadoc for duration criteria.